zeroconf/test.py

```diff
@@ -1368,3 +1400,7 @@ def send(out, addr=r._MDNS_ADDR, port=r._MDNS_PORT):
 
     # unregister
     zc.unregister_service(info)
+
+
+if __name__ == "__main__":
```

Comment:
Let's keep this out – it'd suggest that using the unittest module is a supported way of running project's test, but it's not.

Reply:
Sorry about this creating in again, I’ve stopped using this method myself now anyway!